app/content/analytics-toolbox-redshift/sql-reference/tiler.md

```diff
@@ -0,0 +1,58 @@
+
+| Option | Description |
+| :----- | :------ |
+|`geom_column`| Default: `"geom"`. A `VARCHAR` that indicates the name of the geography column that will be used. The geography colum must be of type `GEOGRAPHY` and contain only points. |
+|`zoom_min`| Default: `0`. An `INTEGER` that defines the minimum zoom level for tiles. Any zoom level under this level won't be generated.|
+|`zoom_max`| Default: `15`. An `INTEGER` that defines the maximum zoom level for tiles. Any zoom level over this level won't be generated. Maximum is 20.|
+|`aggregation_resolution`| Default: `6`. An `INTEGER` that specifies the resolution of the spatial aggregation.<br/><br/>Aggregation for zoom `z` is based on quadkey cells at `z + resolution level`. For example, with resolution `6`, the `z0` tile will be divided into cells that match the `z6` tiles, or the cells contained in the `z10` tile will be the boundaries of the `z16` tiles within them. In other words, each tile is subdivided into `4^resolution` cells, which is the maximum number of resulting features (aggregated) that the tiles will contain.<br/><br/>Note that adding more granularity necessarily means heavier tiles which take longer to be transmitted and processed in the final client, and you are more likely to hit the internal memory limits.|
+|`properties`| Default: {}. A JSON object that defines the extra properties that will be included associated to each cell feature. Each `property` is defined by its name, type (Number, Boolean or String) and formula (any SQL formula that uses an [aggregate function]((https://docs.aws.amazon.com/redshift/latest/dg/c_Aggregate_Functions.html)) supported by Redshift and returns the expected type) to generate the properties from all the values of the points that fall under the cell.|
+
+**Result**
+
+The generated tileset consists of a table with the following columns, where each row represents a tile:
+* `Z`: zoom level of the tile.
+* `X`: X-index of the tile (`0` to `2^Z-1`).
+* `Y`: Y-index of the tile (`0` to `2^Z-1`).
+* `DATA`: contents of the tile, encoded as a GeoJSON string (a feature collection). It will contain the resulting points (location of the aggregated features) and their attributes (as defined by `properties`).
+
+Additionally, there is a row identified with `Z=-1` which contains metadata about the tileset in the `DATA` column in JSON format. It contains the following properties:
+
+* `bounds`: geographical extents of the source as a string in `Xmin, Ymin, Xmax, Ymax` format.
+* `center`: center of the geographical extents as `X, Y, Z`, where the `Z` represents the zoom level where a single tile spans the whole extents size.
+* `zmin`: minimum zoom level in the tileset.
+* `zmax`: maximum zoom level in the tileset.
+* `tilestats`: stats about the feature's properties. In addition to its name (`attribute`) and `type`, it contains `min`, `max`, `average` and `sum`.
+
+{{% customSelector %}}
```

Comment:
Remove `{{% customSelector %}}`

Reply:
I noticed that in other modules like `s2`. Should they be removed as well?

Reply:
Yes, for snowflake and redshift, please